func_env.sh

```diff
@@ -1,6 +1,6 @@
 # loads the .env file in the main directory
 
-commands+=([set-env]="<env_name>:Sets the name of the .env file to be loaded and checks it")
+commands+=([set - env]="<env_name>:Sets the name of the .env file to be loaded and checks it")
```

Comment:
revert this, "set-env" can not have spaces

Reply:
will add quotes to fix